index.js

```diff
@@ -85,11 +85,12 @@ export function mergeTrees(matchingTrees, context) {
 
 class Rule {
 
-    constructor(name, parent, style, filter) {
+    constructor(name, parent, style, filter, properties) {
```

Comment:
Are you talking about this? 

https://github.com/tangrams/unruly/blob/master/index.js#L249